### PR TITLE
fix: Popover option on a searched product are not working correctly(#2k06eqx)

### PR DESCRIPTION
**Pull request by disha1202**

### CODE REVIEW

File: src/views/OrderDetail.vue [hunk 1]
```diff
@@ -327,7 +327,10 @@ export default defineComponent({
           translucent: true,
           showBackdrop: true,
           componentProps: { 'id': id, 'isVirtual': isVirtual, 'item': item }
-        })
+        });
+        popover.onDidDismiss().then(() => {
```
Comment (adityasharma7) [inline src/views/OrderDetail.vue:331]: Why these changes are done?

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/OrderDetail.vue (hunk 1 above)
Comment (disha1202) [reply src/views/OrderDetail.vue:331]: When using revert only option, the data is not updated on searched product.

**Review by adityasharma7 — APPROVED** [review]
